packages/use-contractkit/src/use-contract-kit-methods.ts

```diff
@@ -33,7 +33,9 @@ export function useContractKitMethods(): ContractKitMethods {
           nextConnector.type,
           e
         );
-        dispatch('setConnectorInitError', e);
+        const error =
```

Comment:
For some reason typescript recognized this `catch` parameter to be the type `unknown` which makes sense. Don't know why it didn't catch this before.